includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -569,6 +604,15 @@ public function display_payment_request_button_html() {
 		?>
 		<div id="wc-stripe-payment-request-wrapper" style="clear:both;padding-top:1.5em;display:none;">
 			<div id="wc-stripe-payment-request-button">
+				<?php
+					if ( $this->is_custom_button() ) {
+						$label = $this->get_button_label();
+						$theme = $this->get_button_theme();
+						$height = $this->get_button_height();
+						$style = 'height:' . $height . 'px;';
+						echo "<button id=\"wc-stripe-custom-button\" class=\"button $theme\" style=\"$style\"> $label </button>";
```

Comment:
The values here need to be escaped before being used. `$theme` requires a bit more manipulation, but the `$height` and `$label` values are text fields and might allow someone to directly enter malicious values.

Reply:
Agree, fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1197/commits/15e8021950b7b0b3f8ac189e088cb8f9784b6ba1